### ISSUE TITLE
[hue] Trigger firing multiple times after "GO_AWAY => closing session" event

### ISSUE THREAD

**Issue by triviniu** [openhab/openhab-addons#15630]
## Expected Behavior
The hue binding delivers triggers for certain events, such as button presses on a hue dimmer or hue tap switch. A trigger should fire once per event (e.g. button press or button release). The associated events.log file should look like this for a short dimmer press on button 1: 

events.log before issue:
```
2023-09-17 18:38:28.587 [INFO ] [openhab.event.ChannelTriggeredEvent ] - hue:device:2:Hue_DM_01:button-last-event triggered 1000
2023-09-17 18:38:29.596 [INFO ] [openhab.event.ChannelTriggeredEvent ] - hue:device:2:Hue_DM_01:button-last-event triggered 1002
```

## Current Behavior
After a "closing session" warning seen in the openhab.log, any trigger is delivered multiple times per button press. In the log files, the openhab.log records the following warnings:

```
2023-09-17 20:31:05.326 [WARN ] [e.internal.handler.Clip2ThingHandler] - Command '80.95985916' for thing 'hue:device:2:WhiteAmbianceE27_03', channel 'hue:device:2:WhiteAmbianceE27_03:color-temperature' failed with error 'putResource() error sending request'.
2023-09-17 20:31:05.376 [WARN ] [e.internal.handler.Clip2ThingHandler] - Command '80.95985916' for thing 'hue:device:2:WhiteAmbianceE27_02', channel 'hue:device:2:WhiteAmbianceE27_02:color-temperature' failed with error 'putResource() error sending request'.
2023-09-17 20:31:06.283 [WARN ] [.hue.internal.connection.Clip2Bridge] - Fatal error SessionListenerAdapter GO_AWAY => closing session.
```
and the subsequent trigger in the events.log then all look like: 
```
2023-09-17 20:31:19.824 [INFO ] [openhab.event.ChannelTriggeredEvent ] - hue:device:2:Hue_DM_02:button-last-event triggered 1000
2023-09-17 20:31:19.826 [INFO ] [openhab.event.ChannelTriggeredEvent ] - hue:device:2:Hue_DM_02:button-last-event triggered 1000
2023-09-17 20:31:19.827 [INFO ] [openhab.event.ChannelTriggeredEvent ] - hue:device:2:Hue_DM_02:button-last-event triggered 1002
2023-09-17 20:31:19.827 [INFO ] [openhab.event.ChannelTriggeredEvent ] - hue:device:2:Hue_DM_02:button-last-event triggered 1002

```
meaning they are all twice. This is true for all triggers received through the hue-binding. I have also seen tripled triggers, so presumably this problem can pile up.

Another instance with the same outcome was with a different warning in openhab.log:
```
2023-09-18 02:50:17.529 [WARN ] [e.internal.handler.Clip2ThingHandler] - Command 'OFF' for thing 'hue:device:2:WhiteAmbianceGU10_01', channel 'hue:device:2:WhiteAmbianceGU10_01:brightness' failed with error 'HTTP 2 session is null or closed'.
2023-09-18 02:50:18.527 [WARN ] [.hue.internal.connection.Clip2Bridge] - Fatal error SessionListenerAdapter GO_AWAY => closing session.

```

 
## Possible Solution
A restart of the openhab.service restores the desired behaviour, so I would suspect something is left running after the "closing session" event that should have been closed, and gets cleared with a restart. I have not tried to restart the binding only.
 
## Steps to Reproduce (for Bugs)

Unable, since I do not know how to provoke the reported warnings in the openhab.log

## Context
Since triggers are queued until processed, a triggered rule will fire multiple times, making switching outcomes unpredictable. As a workaround, this behaviour can be "debounced" with additional rules negotiating between the trigger and the targeted rule.

## Your Environment
Most recent openhabian out of the box on RPi 4B 4GB, i.e.:
`Linux openhabian 6.1.21-v8+ #1642 SMP PREEMPT Mon Apr  3 17:24:16 BST 2023 aarch64`
with:
`openHAB 4.0.2 - Release Build`          
and bundle:list output:
`244 │ Active │  80 │ 4.0.2                  │ openHAB Add-ons :: Bundles :: hue Binding`

The RPi and the hue bridge are both on ethernet connected to the same network switch.

**Comment by triviniu** [comment]
PS: The system uses the hue binding in the api v2 configuration.

**Comment by andrewfg** [comment]
After the GO_AWAY the connection instance should be closed and disposed, and then a new connection should be established. Are you saying that this is not the case?

**Comment by andrewfg** [comment]
I am not sure the timing of this issue..

1. There was a PR #15477 ("Resolves issue with duplicate event messages after recycle") included in OH 4.1.0.M1 on September 10th 2023
2. This issue was posted on September 19th 2023 .. so if you had not yet updated to OH 4.1.0.M1 at the time of posting the issue, it is possible that the issue may already have been resolved.

@triviniu => please advise if this issue is resolved.
